editor/src/dialog/dialogs/export_dialog.rs

```diff
@@ -86,6 +86,7 @@ impl PropertyHolder for Export {
 			WidgetHolder::new(Widget::DropdownInput(DropdownInput {
 				selected_index: Some(index as u32),
 				entries,
+				interactive: false,
```

Comment:
What's this for?

Reply:
Do we want it to be true?

Reply:
This is just the artwork bounds dropdown menu, right? My mental model was that the behavior for dropdown menus always are interactive, unless there are specific reasons (like performance) where they really shouldn't be. That way users can learn from that behavior and rely on it everywhere. Unless I'm forgetting something, I don't see why the artwork bounds in this dropdown menu can't be interactive. (If I am forgetting something, please let me know.)